docs/includes/service_input.md

```diff
@@ -0,0 +1,8 @@
+This plugin is a service input. Normal plugins gather metrics determined by the
+interval setting. Service plugins start a service to listens and waits for
+metrics or events to occur. Service plugins have two key differences from
+normal plugins:
+
+1. The agent or plug-in specific interval setting may not apply
```

Comment:
Make the naming more consistent with "Global configuration options" section.
```suggestion
1. The global or plugin specific `interval` setting may not apply
```

Reply:
thx applied